cms/static/sass/_base.scss

```diff
@@ -595,26 +595,27 @@ hr.divide {
 
 .tooltip {
   @include transition(opacity $tmg-f3 ease-out 0s);
-  @include font-size(12);
-  @extend %t-regular;
   @extend %ui-depth5;
   position: absolute;
-  top: 0;
-  left: 0;
-  padding: 0 10px;
-  border-radius: 3px;
-  background: rgba(0, 0, 0, 0.85);
-  line-height: 26px;
-  color: $white;
-  pointer-events: none;
   opacity: 0.0;
+  transform: translateY(-100%);
 
-  &:after {
+  > .tooltip-content {
+    @include font-size(12);
+    @extend %t-regular;
+    line-height: 26px !important;
```

Comment:
Is there some way to avoid using `!important` here? Increase the specificity of the selector, perhaps?